Mathlib/Analysis/Calculus/FDeriv/Equiv.lean

```diff
@@ -355,7 +355,7 @@ theorem HasStrictFDerivAt.of_local_left_inverse {f : E → F} {f' : E ≃L[𝕜]
   · refine (hf.isBigO_sub_rev.comp_tendsto hg).congr' (Eventually.of_forall fun _ => rfl)
       (hfg.mono ?_)
     rintro p ⟨hp1, hp2⟩
-    simp only [(· ∘ ·), hp1, hp2]
+    simp only [(· ∘ ·), hp1, hp2, Prod.map]
```

Comment:
Where does this change come from?

Reply:
I'm using `Prod.map` instead of an inline lambda in some lemma, so it appears in the proof state. `simp` would simplify it but `simp only` needs an extra hint.